pkg/metrics/metrics.go

```diff
@@ -253,7 +254,7 @@ func (c *Config) validate(ctx context.Context) (bool, error) {
 	if err != nil {
 		return false, errors.Wrap(err, "error getting operator namespace")
 	}
-	if wmcoNamespace.Labels["openshift.io/cluster-monitoring"] != "true" {
+	if strings.ToLower(wmcoNamespace.Labels["openshift.io/cluster-monitoring"]) != "true" {
```

Comment:
So this will accept strings like `tRue` etc. Do we want to do that or throw an error?

Reply:
Should be fine IMO. I don't see a reason accept `True` but not `tRue`

Reply:
Thinking more about this, I suggest we rather do [strconv.ParseBool](https://cs.opensource.google/go/go/+/go1.20.2:src/strconv/atob.go;l=10).